Formula/eg-examples.rb

```diff
@@ -0,0 +1,38 @@
+class EgExamples < Formula
+  include Language::Python::Virtualenv
+
+  desc "Useful examples at the command-line"
+  homepage "https://github.com/srsudar/eg"
+  url "https://files.pythonhosted.org/packages/a6/93/38075713a7968a9e8484e894f604f99a68e443e0f9db0ed48063b1241969/eg-1.1.1.tar.gz"
```

Comment:
Since the code is on github, could you use the released .tar.gz file on github? The url is cleaner: `https://github.com/srsudar/eg/archive/v1.1.1.tar.gz`

Reply:
You're right! Sorry for the confusion.